### PR TITLE
Remove extra space at top

### PR DESCRIPTION
**Pull request by knitcodemonkey**
This will have to be corrected in some repos while not in others

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,56 +26,56 @@
...
-    "babel-preset-stage-1": "^6.5.0",
-    "lodash": "^4.15.0",
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.3",
+    "babel-eslint": "^8.2.3",
+    "babel-loader": "^7.1.4",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-1": "^6.24.1",
+    "deepmerge": "^2.1.0",
```
Comment (ryan-roemer) [inline package.json]: Is this needed anymore?

Comment (knitcodemonkey) [reply package.json]: nope. Removed.

File: package.json [hunk 2]
```diff
@@ -4,15 +4,15 @@
...
+    "demo": "webpack-dashboard -- webpack-dev-server --port 5678 --config demo/webpack.config.demo.js --colors --content-base demo --mode=development",
     "lint": "eslint --ext .js,.jsx src",
     "test": "karma start ./karma.conf.js",
     "test-travis": "karma start ./karma.conf.js --singleRun=true",
     "babel": "rimraf lib && babel src/ -d lib/",
     "copy-assets": "rimraf lib/assets/ && cp -r src/assets/ lib/assets/",
     "copy-styles": "rimraf lib/styles/ && postcss src/styles/*.css --verbose --dir lib/styles/",
-    "build": "npm run babel && npm run copy-assets && npm run copy-styles",
-    "prepublish": "npm run build",
+    "build": "npm run babel && npm run copy-assets && npm run copy-styles --mode=production",
```
Comment (ryan-roemer) [inline package.json]: is the `--mode` flag in `npm run copy-styles --mode=production` correctly passed here? I thought for npm you needed to do something like:

```sh
$ npm run copy-styles -- --mode=production
```

Unless they adopted `yarn`'s tweak where you don't need the `--`.

File: package.json [hunk 3]
```diff
@@ -26,56 +26,54 @@
...
+    "eslint-plugin-filenames": "^1.2.0",
+    "eslint-plugin-react": "^7.7.0",
+    "file-loader": "^1.1.11",
+    "image-webpack-loader": "^4.2.0",
+    "json-loader": "^0.5.7",
+    "karma": "^2.0.2",
+    "karma-babel-preprocessor": "^7.0.0",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-mocha": "^1.3.0",
+    "karma-phantomjs-launcher": "^1.0.4",
```
Comment (ryan-roemer) [inline package.json]: We can nuke this!!!

File: webpack.config.test.js [hunk 4]
```diff
@@ -36,14 +36,16 @@ module.exports = {
         ]
       }, {
         test: /\.json$/,
-        loader: "json"
+        exclude: path.resolve(__dirname, "node_modules"), //eslint-disable-line no-undef
```
Comment (ryan-roemer) [inline webpack.config.test.js]: If we can, I would prefer an `include`. But if out of scope for this work, no big deal.

File: webpack.config.test.js [hunk 5]
```diff
@@ -36,14 +36,16 @@ module.exports = {
...
     "jsdom": "window",
-    "react/lib/ExecutionEnvironment": true,
-    "react/lib/ReactContext": true,
-    "react/addons": true
-  }
+    "react/lib/ExecutionEnvironment": "react/lib/ExecutionEnvironment",
+    "react/lib/ReactContext": "react/lib/ReactContext",
+    "react/addons": "react/addons"
+  },
+  mode: 'production'
```
Comment (ryan-roemer) [inline webpack.config.test.js]: NIT: Are we linting out webpack js files I see different styles just in this diff.

File: package.json (hunk 2 above)
Comment (knitcodemonkey) [reply package.json]: I believe they did adopt yarn's tweak, but I can add it in for safety.

Comment (knitcodemonkey) [reply package.json]: Actually, it doesn't need to be there at all.

File: webpack.config.test.js (hunk 4 above)
Comment (knitcodemonkey) [reply webpack.config.test.js]: I'm not 100% sure where it is getting its json files from, so I just made sure to remove node_modules. I'll just remove the exclude for now.

File: webpack.config.test.js (hunk 5 above)
Comment (knitcodemonkey) [reply webpack.config.test.js]: I've added prettier, run lint across the project, and fixed any linting errors I found

File: .eslintrc [hunk 6]
```diff
@@ -1,11 +1,13 @@
...
-  "plugins": [ "react" ],
+  "plugins": ["react", "prettier", "filenames"],
   "rules": {
     "react/jsx-uses-vars": 2,
     "react/jsx-uses-react": 2,
     "react/react-in-jsx-scope": 2,
-    "no-extra-parens": 0
+    "no-extra-parens": 0,
+    "no-empty-label": 0,
+    "filenames/filenames": 0
```
[the remark this answers is not in the document]
Comment (knitcodemonkey) [reply .eslintrc]: no-empty-label no longer exists, but it's still in the walmart project. The no-extra-parens was there when I got here. I was unable to find out why filenames/filenames was not found. I'll remove that rule and let it fail.

```
/Users/jenniferluker/sandbox/formidable-landers/src/index.js
  1:1  error  Rule 'no-empty-label' was removed and replaced by: no-labels  no-empty-label
  1:1  error  Definition for rule 'filenames/filenames' was not found       filenames/filenames
```

File: package.json (hunk 2 above)
Comment (ryan-roemer) [reply package.json]: Are we missing an unpushed commit? Still looks like `--mode=production` here...

**Comment by beccalee123** [comment]
LGTM!

**Pull request merged**